cli/file_copy_getter.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"fmt"
+	"github.com/gruntwork-io/terragrunt/util"
+	"github.com/hashicorp/go-getter"
```

Comment:
NIT: I like to separate out stdlib imports from thirdparty imports from local imports. E.g:

```
import (
  "fmt"
  "net/url"
  "os"

  "github.com/hashicorp/go-getter"

  "github.com/gruntwork-io/terragrunt/util"
)
```

Reply:
https://godoc.org/golang.org/x/tools/cmd/goimports can be used instead of `go fmt`
It works faster
```
terragrunt (go-getter-update*) » time goimports -w cli/file_copy_getter.go
goimports -w cli/file_copy_getter.go  0.01s user 0.00s system 82% cpu 0.015 total
terragrunt (go-getter-update*) » time go fmt cli/file_copy_getter.go
go fmt cli/file_copy_getter.go  0.42s user 0.41s system 244% cpu 0.341 total
```
and automatically manages imports (most of the time) and formats them
```
diff -u cli/file_copy_getter.go.orig cli/file_copy_getter.go
--- cli/file_copy_getter.go.orig	2019-06-12 08:20:02.000000000 +1200
+++ cli/file_copy_getter.go	2019-06-12 08:20:02.000000000 +1200
@@ -2,10 +2,11 @@

 import (
 	"fmt"
-	"github.com/gruntwork-io/terragrunt/util"
-	"github.com/hashicorp/go-getter"
 	"net/url"
 	"os"
+
+	"github.com/gruntwork-io/terragrunt/util"
+	"github.com/hashicorp/go-getter"
 )
```

Reply:
Heh, I use IntelliJ and almost never look at the imports; all imports happen automatically while typing, so the order of them never felt important. That said, we do use `goimports` with Terratest...

Reply:
Just ran `goimports` and it updated 26 files. Probably not worth complicating this PR with that many changes. We should probably just do it in a separate PR and perhaps add it to the `make fmt` command.